openhivenpy/types/member.py

```diff
@@ -75,7 +75,7 @@ async def kick(self) -> bool:
         """
 
         #DELETE api.hiven.io/houses/HOUSEID/members/MEMBERID
-        res = requests.delete(f"https://api.hiven.io/v1/houses/{self._house_id}/members/{self._user_id}", headers={"Content-Type": "application/json", "Authorization": self._TOKEN})
+        res = await self._http_client.delete(f"https://api.hiven.io/v1/houses/{self._house_id}/members/{self._user_id}", headers={"Content-Type": "application/json", "Authorization": self._TOKEN})
         if not res.response_code == 204: #Why not continue using 200 instead of using 204 i have no idea.
```

Comment:
Also, because I noticed this 204 is basically no data at all. That means we got a successful response, but it has no code or response data

Reply:
To that, we need to change this because response_code only works with requests. Now it's called status, yes I know very creative

Reply:
_??????_